convert2rhel/data/8/x86_64/configs/centos-8-x86_64.cfg

```diff
@@ -10,7 +10,7 @@ gpg_fingerprints = 05b555b38483c65d
 excluded_pkgs =
   centos-logos
   centos-indexhtml
-  centos-release*
+  centos*release*
```

Comment:
@bocekm maybe more strict globbing. I am not sure how do you read the config, but what about something like:
```bash
centos-{,linux-}release
```

Reply:
I'll check if such a globbing is supported by dnf.
We would need to change it also [for Oracle Linux](https://github.com/oamg/convert2rhel/blob/main/convert2rhel/data/8/x86_64/configs/oracle-8-x86_64.cfg#L14) to stay consistent.
I'm not sure if making it strict is necessary since in the end we want to get rid of all the centos packages. I wouldn't mind even `centos-*` :)

Reply:
I used your suggested globbing and added the missing centos-repos. And centos-obsolete-packages.

Reply:
Unfortunately, this kind of globbing does not work with dnf API filtering: 
https://github.com/oamg/convert2rhel/blob/416c45574b5c4560d9a2b2afbae9b8895bfa0920/convert2rhel/pkghandler.py#L243
Even though, per docs, it should: https://dnf.readthedocs.io/en/latest/api_queries.html#dnf.query.Query.filter

```
centos-logos{,-httpd} ........................... 0
centos-indexhtml ................................ 0
centos-{,linux-}release ......................... 0
centos-{,linux-}repos ........................... 0
```

Reply:
Filed a bug for dnf https://bugzilla.redhat.com/show_bug.cgi?id=1913418. In the meantime, we'll need to stick with "*".